VMBackup/main/common.py

```diff
@@ -95,6 +95,7 @@ class CommonVariables:
     error_http_failure = 15
     FailedHandlerGuestAgentCertificateNotFound = 16
     #error_upload_status_blob = 16
+    FailedSafeFreezeBinaryNotFound = 18
```

Comment:
this code is reserved for FailedHandlerSnapshotBitlocker in windows. Please re-check and assign a non-conflicting error number. Better to be in the range [121-130]

Reply:
yeah have checked it, modified to 19 will push it along with the other changes in new iteration.
Would also require modification at the CRP side to include this error cde.